go.mod

```diff
@@ -4,17 +4,16 @@ go 1.13
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
```

Comment:
Maybe run `go mod tidy` to clean up dependencies.

Reply:
Good catch, will do